docs/whats-new.asciidoc

```diff
@@ -4,112 +4,140 @@
 
 Here are the highlights of what’s new and improved in {elastic-sec}. For detailed information about this release, check out the <<release-notes, Release notes>>.
 
-Other versions: {security-guide-all}/8.1/whats-new.html[8.1] | {security-guide-all}/8.0/whats-new.html[8.0] | {security-guide-all}/7.17/whats-new.html[7.17] | {security-guide-all}/7.16/whats-new.html[7.16] | {security-guide-all}/7.15/whats-new.html[7.15] | {security-guide-all}/7.14/whats-new.html[7.14] | {security-guide-all}/7.13/whats-new.html[7.13] | {security-guide-all}/7.12/whats-new.html[7.12] | {security-guide-all}/7.11/whats-new.html[7.11] | {security-guide-all}/7.10/whats-new.html[7.10] |
+Other versions: {security-guide-all}/8.2/whats-new.html[8.2] | {security-guide-all}/8.1/whats-new.html[8.1] | {security-guide-all}/8.0/whats-new.html[8.0] | {security-guide-all}/7.17/whats-new.html[7.17] | {security-guide-all}/7.16/whats-new.html[7.16] | {security-guide-all}/7.15/whats-new.html[7.15] | {security-guide-all}/7.14/whats-new.html[7.14] | {security-guide-all}/7.13/whats-new.html[7.13] | {security-guide-all}/7.12/whats-new.html[7.12] | {security-guide-all}/7.11/whats-new.html[7.11] | {security-guide-all}/7.10/whats-new.html[7.10] |
 {security-guide-all}/7.9/whats-new.html[7.9]
 
 // NOTE: The notable-highlights tagged regions are re-used in the Installation and Upgrade Guide. Full URL links are required in tagged regions.
 // tag::notable-highlights[]
 
 [discrete]
-[[features-8.2]]
+[[term-changes-8.3]]
+== Terminology changes
+
+*"Endpoint Security integration" has been renamed to "Endpoint and Cloud Security integration"*
+
+Due to the launch of https://www.elastic.co/security/cloud-security[Elastic Security for Cloud], the *Endpoint Security integration*, which allows the {agent} to monitor for events on your host, has been renamed to *Endpoint and Cloud Security integration*. Please note that general industry term references to endpoint security have not changed.
+
+[role="screenshot"]
+image::whats-new/images/8.3/cloud-integration.png[]
 
 [discrete]
-== New landing pages added to the left navigation menu
+[[features-8.3]]
 
-Several new landing pages were added to the navigation menu in 8.2:
 
-The *Getting started* page provides guidance on adding data to your environment. When new users install {elastic-sec}, this page is now the default view.
+[discrete]
+== New streamlined navigation
+
+An optional, new navigation menu, which can be enabled in the {security-guide}/advanced-settings.html#_enable_grouped_navigation[advanced {kib} settings], groups related pages and highlights commonly visited areas for a streamlined experience.
 
 [role="screenshot"]
-image::whats-new/images/8.2/getting-started.png[Getting started landing page]
+image::whats-new/images/8.3/grouped-nav.png[Grouped navigation menu]
```

Comment:
Image quality for this screenshot looks low-res -- could we use a direct copy of the version at `docs/getting-started/images/grouped-nav-ui.png`? It looks like the file might have gotten saved from high-res (Retina) to lower-res when it was copied into the whats-new folder.

Or just use the original image reference from the docs -- I checked this locally and it should resolve to the same image, same dimensions:
```suggestion
image::getting-started/images/grouped-nav-ui.png[width=75%][height=75%][Grouped navigation menu]
```